internal/app/referenceclient/client.go

```diff
@@ -227,7 +227,7 @@ func invoke(ctx context.Context, req *conformancev1.ClientCompatRequest, referen
 		transport = &contextFixTransport{http3.RoundTripper{
 			DisableCompression: true,
 			TLSClientConfig:    tlsConf,
-			QuicConfig:         &quic.Config{MaxIdleTimeout: 20 * time.Second, KeepAlivePeriod: 5 * time.Second},
+			QUICConfig:         &quic.Config{MaxIdleTimeout: 20 * time.Second, KeepAlivePeriod: 5 * time.Second},
```

Comment:
This is documented as one of the breaking changes in v0.43.0: https://github.com/quic-go/quic-go/releases/tag/v0.43.0